libs/micronaut-worker/src/main/java/com/agorapulse/worker/annotation/Fork.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2022 Agorapulse.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.agorapulse.worker.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Sets the number of jobs running in parallel on a one server.
+ */
```

Comment:
its the maximum number of jobs that can run in parallel, did I understand correctly?

Reply:
No, this option is really about defining how many jobs should start in parallel. E.g. `@Fork(2) @FixedRate('1s')` will run the job twice for every specified configuration (e.g. the method will be called twice every `1s`). But for consumer jobs all the other constraints apply - so for example if there are no messages in the queue then the job is not triggered at all.